airqo-platform/AirQo-api
# Update: analytics data download

- What does this PR do?
This PR provide some improvement to the data download endpoint
- What JIRA task is related to this PR?
Checkout JIRA card [here](https://airqoteam.atlassian.net/browse/NLYTCS-104)
- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/analytics`. Set up Python environment: `python3 -m venv env` (if not yet created), then activate it using this command: `source env/bin/activate`.
Then run `pip install -r requirements.txt` to get all the necessary package. Afterwards, run `flask run` to test out the endpoints using any REST client tool available to you.
Which endpoints should be/are ready for testing?
base URL: `http://localhost:5000 `
[POST] /api/v1/data/download?type=csv
[POST] /api/v1/data/download?type=json
Take a look at the [API](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=1469478754) documentation for more details.
